worker.go

```diff
@@ -156,6 +156,12 @@ func buildManagedModel(certsProvider certs.CertificatesProvider, ingresses []*ku
 				log.Printf("failed to find a certificate for %v: %v", ingress.CertHostname(), err)
 				continue
 			}
+		} else { // validate that certificateARN exists
+			err := checkCertificate(certsProvider, certificateARN)
+			if err != nil {
+				log.Printf("Failed to validate certificate: %v", err)
```

Comment:
Better message would be:

    "Failed to find certificate with ARN %s: %v", arn, err

Reply:
I have updated the error messages.